book.toml

```diff
@@ -82,3 +82,22 @@ line-numbers = true
 
 [output.exerciser]
 output-directory = "comprehensive-rust-exercises"
+
+[output.i18n]
+default_language = "en"
+translate_all_languages = true
+optional = true
+
+[output.i18n.languages]
+"en" = "English"
+"es" = "Spanish (Español)"
+"ko" = "Korean (한국어)"
+"pt-BR" = "Brazilian Portuguese (Português do Brasil)"
+
+[output.tera-backend]
+after = "html"
```

Comment:
Doesn't do anything. I will remove this. The order is actually alphabetical. I will open a PR on mdbook to document it when I have time